### ISSUE TITLE
VS Code feature request: Show code running buttons in code cell

### ISSUE THREAD

**Issue by mine-cetinkaya-rundel** [quarto-dev/quarto#281]
In the source editor in VS Code, the button to run the code in a cell is visible whether the cursor is in the cell or not.

<img width="570" alt="Screenshot 2023-09-27 at 11 47 50 PM" src="https://github.com/quarto-dev/quarto/assets/5965649/32510fe7-c740-4272-af5b-ef987e949d06">

However, in the visual editor cursor must be in the cell for those buttons to be visible.

<img width="662" alt="Screenshot 2023-09-27 at 11 48 36 PM" src="https://github.com/quarto-dev/quarto/assets/5965649/188d1061-4005-4005-ad63-02176b7f7a19">

I think this is confusing because if you're not familiar with code cells, you might not know to click on them to reveal the buttons. I think the buttons should either always be visible, or at a minimum become visible when hovering over the cells (without the need to click on them).

**Comment by jjallaire** [comment]
I agree that this isn't great. The reason we do this is to workaround more focus issues -- right now if the user clicks one of those buttons it doesn't drive selection into the editor (which means that the run button doesn't work as expected). We will also try to remedy this but not right now.
